From 803cf2a5192502760e8563a3ef66ccb712d594cc Mon Sep 17 00:00:00 2001
From: Vitaly Lipatov <lav@etersoft.ru>
Date: Mon, 5 Aug 2013 13:55:05 +0400
Subject: [PATCH 21/25] 0.16-eter23.svn724
To: wine-patches <wine-patches@winehq.org>
Reply-To: wine-devel <wine-devel@winehq.org>

- add Num Lock state as a parameter to be send
---
 .gear/opennx.spec | 1 -
 1 file changed, 1 deletion(-)

diff --git a/.gear/opennx.spec b/.gear/opennx.spec
index 35613df..00c80f5 100644
--- a/.gear/opennx.spec
+++ b/.gear/opennx.spec
@@ -92,7 +92,6 @@ install -m 644 etc/*.rules %buildroot%_sysconfdir/udev/rules.d
 %endif
 
 %files -f %name.lang
-%doc COPYING INSTALL
 %_bindir/%name
 %_libdir/%name
 %_datadir/%name
-- 
1.8.4.3

